types/container/host_config.go

```diff
@@ -256,6 +267,8 @@ type Resources struct {
 	OomKillDisable       *bool           // Whether to disable OOM Killer or not
 	PidsLimit            int64           // Setting pids limit for a container
 	Ulimits              []*units.Ulimit // List of ulimits to be set in the container
+	Priorities           []*IfPriority   // priority of network traffic for container
+	ClassID              uint32          // class identifier for container's network packets
```

Comment:
I think there should be some prefix string for understandability.
e.g.
`NetPriorities`, `NetClassID`